Terminal.Gui/Views/TableView/TableView.cs

```diff
@@ -121,6 +121,11 @@ public class TableView : View {
 			}
 		}
 
+		/// <summary>
+		/// The minimum number of characters to render in any given column.
+		/// </summary>
+		public int MinCellWidth { get; set; }
```

Comment:
This already exists in ColumnStyle.  See: `ColumnStyle.MinWidth`

Please can you look into setting the column style rather than adding a new property to `TableView`?

```csharp
for (int i = 0; i < Columns; i++) {
    tv.Style.GetOrCreateColumnStyle (i).MinWidth = 1;
}
```

Reply:
> This already exists in ColumnStyle.  See: `ColumnStyle.MinWidth`

I was aware of this, but you don't think it's nice to have a global min width for the entire table just like the global max regardless of the source?

Reply:
> > This already exists in ColumnStyle.  See: `ColumnStyle.MinWidth`
> 
> I was aware of this, but you don't think it's nice to have a global min width for the entire table just like the global max regardless of the source?

Ok that makes sense, seems handy.  Thanks for implementing this.

Next time though it would simplify review if it were a separate pull request with its own unit test(s) and scenario support in the main TableEditor scenario e.g. a PR "Adds MinCellWidth to TableView".